manifest.json

```diff
@@ -48,6 +51,12 @@
       "run_at": "document_start",
       "all_frames": false,
       "js": ["content/install.js"]
+    },
+    {
+      "matches": ["<all_urls>"],
```

Comment:
eeeh! please use your old `matches` with *.user.css suffix. We don't need more than one content script on the majority of pages.